.gitignore

```diff
@@ -8,6 +8,9 @@
 .project
 .settings
 .classpath
+bin/
+.factorypath
+
```

Comment:
What is this about ? Is this commit needed ?

Reply:
"_Needed_" is strong language, but it's _nice to have_ for a dinosaur using eclipse like me :) I'd be happy to make this a separate PR is that would help though

Reply:
Nah it's good for this small addition.